src/main/webapp/app/exercise/result/result.utils.ts

```diff
@@ -274,7 +274,7 @@ export const isOnlyCompilationTested = (result: Result | undefined, templateStat
  *
  * @return {string} the css class
  */
-export const getTextColorClass = (result: Result | undefined, templateStatus: ResultTemplateStatus) => {
+export const getTextColorClass = (result: Result | undefined, participation: Participation, templateStatus: ResultTemplateStatus) => {
```

Comment:
Does participation really always need to be defined, why don't we allow undefined?